Lib/test/test_os.py

```diff
@@ -1295,7 +1294,7 @@ def walk(self, top, **kwargs):
             bdirs[:] = list(map(os.fsencode, dirs))
             bfiles[:] = list(map(os.fsencode, files))
 
-@unittest.skipUnless(hasattr(os, 'fwalk'), "Test needs os.fwalk()")
+
```

Comment:
I don't think you need to remove this, since the `os` module *does* define fwalk on some unixes, but this won't necessarily work when `os` doesn't define fwalk (i.e. when `not ({open, stat} <= supports_dir_fd and {listdir, stat} <= supports_fd)`)

Reply:
Yes, this line should be kept as is.